demo/server_config.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I have an aversion against hard-coded paths or required node modules. I definitely want to make sure that the demo app runs flawlessly even if these prerequisites are not available on a developer machine. Maybe with a hint in the ToolConsole on how to test additional tools with the MCP server (install node and relevant node-modules ...)